root/etc/confd/templates/www.conf.tmpl

```diff
@@ -217,7 +217,6 @@ catch_workers_output = yes
 ;                specified at startup with the -d argument
 ;php_admin_value[sendmail_path] = /usr/sbin/sendmail -t -i -f www@my.domain.com
 php_flag[display_errors] = off
-;php_admin_value[error_log] = /opt/rh/php55/root/var/log/php-fpm/www-error.log
```

Comment:
This right here... if we could get this to work with a similarly generic path, we could move the stdout symlink to base as well and not manage it as a PHP-version variation. That's aside the point of this PR, just excited to see it.